src/components/Dashboard/Flavors.js

```diff
@@ -0,0 +1,81 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { Table } from 'react-bootstrap';
+import { Layout } from '.';
+
+import { actions as flavorsActions } from 'reducers/flavors';
+import { getAllFlavors } from 'selectors/flavors';
+
+export class Flavors extends Component {
+  static propTypes = {
+    actions: PropTypes.object.isRequired,
+    opt: PropTypes.object.isRequired,
+    flavors: PropTypes.array
+  };
+  constructor(props) {
```

Comment:
No need for the constructor if all you're doing is calling `super`

Reply:
Removed